libs/SmartSync/src/com/salesforce/androidsdk/smartsync/target/LayoutSyncDownTarget.java

```diff
@@ -39,12 +39,13 @@
 
 /**
  * Sync down target for object layouts. This uses the '/ui-api/layout' API to fetch object layouts.
+ * The easiest way to use this sync target is through {@link com.salesforce.androidsdk.smartsync.manager.LayoutSyncManager}.
  *
  * @author bhariharan
  */
 public class LayoutSyncDownTarget extends SyncDownTarget {
 
-    public static final String SOBJECT_TYPE = "sobjectType";
+    public static final String OBJECT_TYPE = "objectType";
```

Comment:
We are using sobjectType in MRUSyncDownTarget - should we stay consistent with that?

Reply:
Sure - I can change that. I changed it to match the `describe` response. I think that's `objectType`.